jdk9/README.md

```diff
@@ -1,32 +1,5 @@
 # Java JDK9
 
-## Description
+This plan has been deprecated as it has been marked EOL by upstream.
 
-This package contains the Java JDK 9 to compile java applications.
-
-Include this as a dependency in your own plan, so you can build your applications / jar files.
-
-Couple it with the `core/jre9` package for running your app (if necessary)
-
-## Usage
-
-Your `plan.sh` dependencies will look similar to this:
-
-```
-pkg_deps=(
-  core/jre9
-)
-pkg_build_deps=(
-  core/jdk9
-)
-```
-
-Build your application with:
-
-```
-do_build() {
-  javac ...
-}
-```
-
-Then, from your own `plan.sh` run hook, you can `exec java ...` to run your application (requires `core/jre9`).
+https://www.oracle.com/technetwork/java/javase/9-0-4-relnotes-4021191.html
```

Comment:
Can we still leave the package description for those that choose to use a deprecated plan?

First thing in the README would be the deprecation notice. Bolded, or emphasised how you like, but I think retaining the readme information is useful.

Reply:
🤔 The only concern I would have with that is that there is the implication that this Plan is still in some way functional, despite being deprecated. The upstream has pulled all sources so it can't be rebuilt, and our deprecation RFC calls for toggling all packages private so they are no longer visible (unless you have the fully qualified package identifier). 

I'm not opposed to including existing README below the deprecation notice (and would update the other Deprecated plans for consistency)  but I'd like to get your thoughts on the above.

Reply:
Oh. Thats a particularly interesting case given its no longer able to be rebuilt.
I'm going to resolve my review and approve. I think this is OK 👍 

Thanks for the discussion @smacfarlane